### PR TITLE
ItemGroup changes

### PR DESCRIPTION
**Pull request by ybw0014**
- Add sound for ItemGroups (same as other Slimefun ItemGroup)
- Use `ChestMenuUtils.getEmptyClickHandler()` to replace all empty handlers `(p, slot, itemStack, clickAction) -> false`.
- Change back button's lore to match slimefun core's back buttons

### CODE REVIEW

File: src/main/java/io/github/sefiraat/emctech/slimefun/groups/EmcSlimefunDictionaryFlexGroup.java [hunk 1]
```diff
@@ -59,20 +60,21 @@ public boolean isVisible(Player player, PlayerProfile playerProfile, SlimefunGui
 
     @Override
     @ParametersAreNonnullByDefault
-    public void open(Player player, PlayerProfile profile, SlimefunGuideMode mode) {
+    public void open(Player p, PlayerProfile profile, SlimefunGuideMode mode) {
```
Comment (Sefiraat) [inline src/main/java/io/github/sefiraat/emctech/slimefun/groups/EmcSlimefunDictionaryFlexGroup.java:63]:
```suggestion
    public void open(Player player, PlayerProfile profile, SlimefunGuideMode mode) {
```

**Review by Sefiraat — APPROVED** [review]

File: src/main/java/io/github/sefiraat/emctech/slimefun/groups/EmcSlimefunDictionaryFlexGroup.java (hunk 1 above)
Comment (ybw0014) [reply src/main/java/io/github/sefiraat/emctech/slimefun/groups/EmcSlimefunDictionaryFlexGroup.java:63]: this was changed to `p` for consistency between `open` methods of flex groups

**Comment by ybw0014** [comment]
And why it is an approval with suggestion

**Comment by ybw0014** [comment]
I need to revert that commit

**Review by Sefiraat — APPROVED** [review]

**Pull request merged**